kogito-usertasks-custom-lifecycle-springboot/README.md

```diff
@@ -12,9 +12,29 @@ This example shows
 
 * working with user tasks
 * four eye principle with user tasks
-	
-	
+
```

Comment:
In addition this quickstart also shows how to use custom life cycle for user tasks that uses custom life cycle phases compared to those supported by default.

- Start
- Complete - extended the default one that allows only to complete started tasks

Reply:
fixed @radtriste